source/frontend/src/components/common/form/ToggleSaveInput/ToggleSaveInputView.tsx

```diff
@@ -65,9 +66,20 @@ export const ToggleSaveInputView: React.FC<IToggleSaveInputViewProps> = ({
   } else {
     return (
       <>
-        {asCurrency ? formatMoney(Number(value)) : value}
-        <EditButton onClick={() => setIsEditing(true)} />
+        <StyledCompensationContainer>
+          {asCurrency ? formatMoney(Number(value)) : value}
+          <EditButton onClick={() => setIsEditing(true)} />
+        </StyledCompensationContainer>
```

Comment:
**nit:** This component is generic so the styled container shouldn't be named "StyledCompensationContainer" - maybe just "StyledContainer". This is very minor of course